pkgs/development/python-modules/beautifulsoup4/default.nix

```diff
@@ -25,7 +25,7 @@ buildPythonPackage rec {
   };
 
   nativeBuildInputs = [
-    sphinxHook
+    python.pythonForBuild.pkgs.sphinxHook
```

Comment:
the hook splicing got broken in https://github.com/NixOS/nixpkgs/pull/194205

Reply:
*sigh*

Reply:
we can still merge this but with a comment or a mention in the commit msg to 24dd69bac00c0b5c1c16ad0fa3e32ca0d39bd60c